Casks/affinity-photo.rb

```diff
@@ -0,0 +1,23 @@
+cask "affinity-photo" do
+  version "1.9.3"
+  sha256 :no_check
+
+  url "https://store.serif.com/download/075b51/"
+  name "Affinity Photo"
+  desc "Professional Image Editing Software"
```

Comment:
Minor style change to match other casks where (for `desc` stanza) we capitalize first word and proper nouns like a normal sentence (as opposed to `name` stanza which we capitalize like a title)
```suggestion
  desc "Professional image editing software"
```

Reply:
Gotcha, will keep in mind 👍